src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/myDatabase?useSSL=false&serverTimezone=UTC";
```

Comment:
The `DB_URL` is incomplete and needs to be a full JDBC URL. For example, it should look like `jdbc:mysql://localhost:3306/yourDatabaseName?serverTimezone=UTC`, where `yourDatabaseName` is the name of your database.